src/styles/main.scss

```diff
@@ -10,17 +10,78 @@ body {
   font-weight: 900;
 }
 
+.move {
+  &-up {
+    &--1 {
+      transform: translateY(-85px);
+    }
+    &--2 {
+      transform: translateY(-170px);
+    }
+    &--3 {
+      transform: translateY(-255px);
+    }
+  }
+
+  &-down {
+    &--1 {
+      transform: translateY(85px);
+    }
+    &--2 {
+      transform: translateY(170px);
+    }
+    &--3 {
+      transform: translateY(255px);
+    }
+  }
+
+  &-left {
+    &--1 {
+      transform: translateX(-85px);
+    }
+    &--2 {
+      transform: translateX(-170px);
+    }
+    &--3 {
+      transform: translateX(-255px);
+    }
+  }
+
+  &-right {
+    &--1 {
+      transform: translateX(85px);
+    }
+    &--2 {
+      transform: translateX(170px);
+    }
+    &--3 {
+      transform: translateX(255px);
+    }
+  }
+}
+
 .field-cell {
   background: #d6cdc4;
+  padding: 0;
   width: 75px;
   height: 75px;
   border-radius: 5px;
   color: #776e65;
-  box-sizing: border-box;
+  box-sizing: content-box;
```

Comment:
The `box-sizing: content-box;` is typically not recommended unless you have a specific reason for using it. Most modern CSS frameworks use `box-sizing: border-box;` by default for a more intuitive sizing model.